### PR TITLE
Prefer public repositories

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/haskell/hackage-server/issues/1222

### CODE REVIEW

File: datafiles/templates/upload.html.st [hunk 1]
```diff
@@ -93,6 +93,22 @@ service.</p>
 material that you upload. For example, check that your tarball includes the
 license files of any 3rd party code that you include.</p>
 
+<h3>Public repositories</h3>
+
+It is preferable if any web pages linked from the following fields are publicly accessible:
+<ul>
+<li> Bug report URL
```
Comment (ffaf1) [inline datafiles/templates/upload.html.st:100]: I suspect: missing `</li>`.

Comment (glyn) [reply datafiles/templates/upload.html.st]: Fixed, thanks. (Too used to markdown these days.)

**Comment by gbaz** [comment]
looks good, ty!

**Comment by glyn** [comment]
Thanks for merging, @gbaz. https://hackage.haskell.org/upload hasn't been updated, so does some manual process need to be run?

**Comment by gbaz** [comment]
Yeah, docs are updated on redeploys, which are irregular, unless there's a burning need :-)

**Pull request merged**